src/System.Collections/src/System/Collections/Generic/LinkedList.cs

```diff
@@ -8,20 +8,40 @@
 
 namespace System.Collections.Generic
 {
+    /// <summary>
+    /// A circular doubly linked list.
```

Comment:
@stephentoub Yep, there's a plan in place.  We're currently blocked on a [Roslyn bug](https://github.com/dotnet/roslyn/issues/7829), but once that's fixed we think we can do the conversion pretty soon.